backend/tests/test_chmeme_rec.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import torch
+from transformers import AutoModel
+import requests
+from sklearn.metrics.pairwise import cosine_similarity
+from PIL import Image
+
+
+class ch_meme:
```

Comment:
You need to use `Camel-Case` for the name of class in Python, such as `ChineseMemeRecommender`